addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/ExtensionOpenApi.java

```diff
@@ -41,6 +41,9 @@
 import org.parosproxy.paros.model.Session;
 import org.parosproxy.paros.network.HttpSender;
 import org.parosproxy.paros.view.View;
+import org.zaproxy.addon.commonlib.ExtensionCommonlib;
```

Comment:
Should `ExtensionCommonlib` be set as a dependency within `ExtensionOpenApi`? It works as-is, but for correctness sake should this be setting dependencies constant and adding an overridden `getDependencies` method that returns it?

Reply:
Yes, otherwise as is it will lead to `NullPointerException`.